src/sdk/bucketspace_fs.js

```diff
@@ -345,7 +340,7 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             _id: mongo_utils.mongoObjectId(),
             name,
             tag: js_utils.default_value(tag, undefined),
-            owner_account: account._id,
+            owner_account: account.owner ? account.owner : account._id,
```

Comment:
can you add a comment why the owner of the bucket is the account owner and not the user? where is this behavior documented?